### PR TITLE
Add `notApproved` tier for closed beta

### PR DESCRIPTION
**Pull request by johnhaley81**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -321,7 +321,7 @@ export const TwitterFeedbackWithMaybeAnalysisSchema = TwitterFeedbackSchema.keys
   .required();
 
 // currently only 1 tier
```
Comment (smith-kyle) [inline src/index.js]: Can you delete this comment?

**Review by smith-kyle — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (johnhaley81) [reply src/index.js]: Bah, yeah I can do that.

Comment (johnhaley81) [reply src/index.js]: Done.

**Comment by smith-kyle** [comment]
and you have a conflict with dist/index.js

**Pull request merged**